src/stores/account.ts

```diff
@@ -245,4 +250,23 @@ export default class AccountStore implements IAccountStore {
     this._privateKeyToRecovery = null;
     return account;
   };
+
+  fixKeyLength = async (v: ProtectedPrivateKey, passphrase: string) => {
```

Comment:
I believe that leaving comments here to guess the context(i.e., what is the "fix"), for future readers would be nice.

Reply:
Hmm... that what could be the better way? that;
1. Change function name to `padShortKey` and leave no comment
2. Comment what it does.

Reply:
`padShortKey` seems nice. but I guess comments might be still useful because it's too complicate and odd situation to cover naming only. 😅